feat: LSP vtsls (#1)
sublimelsp/LSP-vtsls: README.md

# LSP-vtsls
VSCode's TypeScript and JavaScript support for Sublime's LSP plugin provided through [vtsls](https://github.com/yioneko/vtsls).

## What is vtsls ?

> This is an LSP wrapper around TypeScript extension bundled with VSCode. All features and performance are nearly the same. you can enjoy VSCode's typescirpt features into Sublime Text.

Reviewer: ```suggestion
This is an LSP wrapper around TypeScript extension bundled with VSCode. All features and performance are nearly the same. You can enjoy VSCode's TypeScript features within Sublime Text.
```

## Installation

 * Install [`LSP`](https://packagecontrol.io/packages/LSP) and `LSP-vtsls` from Package Control.

>  * For ST4: The TypeScript and React (TSX) syntaxes are built-in so no need to install anything else.

Reviewer: Remove this line.
You meant to keep the ` * Restart Sublime.` line.

## Configuration

Open the configuration file using the Command Palette `Preferences: LSP-vtsls Settings` command or open it from the Sublime menu.

> this plugin is a wrapper for vtsls so all its configrations are supported here. you can see [configration schema](https://github.com/yioneko/vtsls/blob/main/packages/service/configuration.schema.json) and customize as you want by providing those values in setting by `Preferences: LSP-vtsls Settings`.

Reviewer: ```suggestion
This plugin is a wrapper for vtsls so all its configrations are supported here. You can see [configuration schema](https://github.com/yioneko/vtsls/blob/main/packages/service/configuration.schema.json) and customize as you want by providing those values in `Preferences: LSP-vtsls Settings`.
```

# LSP-vtsls
VSCode's TypeScript and JavaScript support for Sublime's LSP plugin provided through [vtsls](https://github.com/yioneko/vtsls).

## What is vtsls ?
This is an LSP wrapper around TypeScript extension bundled with VSCode. All features and performance are nearly the same. you can enjoy VSCode's typescirpt features into Sublime Text.

## Installation

>  * Install [`LSP`](https://packagecontrol.io/packages/LSP) and `LSP-typescript` from Package Control.

Reviewer 2: You probably missed to change `LSP-typescript to `LSP-vtsls`

 * For ST4: The TypeScript and React (TSX) syntaxes are built-in so no need to install anything else.

>  * For ST3: If you are working with TypeScript install [TypeScript Syntax](https://packagecontrol.io/packages/TypeScript%20Syntax). If you are working with React install [JSCustom](https://packagecontrol.io/packages/JSCustom).

Reviewer 2: You can remove this section `* For ST3: If you are working with TypeScript install [` As this package is no longer ST3 compatible.

## Installation

 * Install [`LSP`](https://packagecontrol.io/packages/LSP) and `LSP-typescript` from Package Control.
 * Install [`LSP`](https://packagecontrol.io/packages/LSP) and `LSP-vtsls` from Package Control.
 * For ST4: The TypeScript and React (TSX) syntaxes are built-in so no need to install anything else.
 * For ST3: If you are working with TypeScript install [TypeScript Syntax](https://packagecontrol.io/packages/TypeScript%20Syntax). If you are working with React install [JSCustom](https://packagecontrol.io/packages/JSCustom).

>  * Restart Sublime.

Reviewer 2: You accidentally deleted the last step. А restart might be required
